registration.css

```diff
@@ -0,0 +1,21 @@
+.personalInfo{
+	.margin top: 20px;
+	padding: 10px;
+	margin: 20px;
```

Comment:
You don't have `.margin` class, if you wanna set top margin you need to use following rule
`margin-top: 20px;`